### PR TITLE
Included PR and issue templates

### PR DESCRIPTION
**Pull request by chandrams**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Fixes # (issue)
+
+### Type of change
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Docs update
+- [ ] Breaking change (What changes might users need to make in their application due to this PR?)
+- [ ] Requires DB changes
+
```
[the remark this answers is not in the document]
Comment (chandrams) [reply .github/pull_request_template.md:17]: Done

**Review by khansaad — APPROVED** [review]
LGTM!

**Comment by dinogun** [comment]
@chandrams I see that github is saying we are using an older version of the issue template, can we update it

**Comment by chandrams** [comment]
@dinogun - I have updated the issue templates, please review, thank you.

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**